pod/perlhacktips.pod

```diff
@@ -1695,6 +1695,8 @@ determine whether to log the event, and if so how:
 
     $ENV{PERL_MEM_LOG} =~ /m/           Log all memory ops
     $ENV{PERL_MEM_LOG} =~ /s/           Log all SV ops
+    $ENV{PERL_MEM_LOG} =~ /c/           Additionally log C backtrace for
+                                        new_SV events
```

Comment:
How about how about `$ENV{PERL_MEM_LOG} =~ /c(\d*)/` to log the specified number of levels of backtrace (default 3)?

Reply:
I could add that but that'll complicate the sanitisation code in `handy.h` relating to it.

Reply:
I agree with Leonerd

Reply:
Fair enough, that can be a future enhancement if someone feels a strong need.